fabric-networking-v0/src/main/java/net/fabricmc/fabric/impl/networking/PacketDebugOptions.java

```diff
@@ -18,6 +18,7 @@
 
 public final class PacketDebugOptions {
 	public static final boolean DISABLE_BUFFER_RELEASES = System.getProperty("fabric.networking.broken.disableBufferReleases", "false").equalsIgnoreCase("true");
+	public static final boolean WARN_UNREGISTERED_PACKETS = System.getProperty("fabric.networking.warnUnregisteredPackets", "true").equalsIgnoreCase("true");
```

Comment:
@sfPlayer1 If I change the default value to `Logger.isDebugEnabled()` call instead of `true`, would it be better, or is there any other concerns?